packages/ag-charts-website/src/content/menu/data.json

```diff
@@ -0,0 +1,352 @@
+                    { "title": "Events", "path": "charts-events" }
+                ]
+            },
+            {
+                "title": "Layout & Styling",
+                "items": [
+                    { "title": "Layout", "path": "charts-layout" },
+                    { "title": "Themes", "path": "charts-themes" },
+                    { "title": "Overlays", "path": "charts-overlays" },
+                    { "title": "Background", "path": "charts-background" },
+                    {
+                        "title": "Background Image",
+                        "path": "charts-background-image",
+                        "isEnterprise": true
+                    }
+                ]
+            },
+            {
+                "title": "Axes",
+                "items": [
+                    { "title": "Axis Types", "path": "charts-axes-types" },
+                    { "title": "Axis Ticks", "path": "charts-axes-ticks" },
+                    { "title": "Axis Domain", "path": "charts-axes-domain" },
+                    { "title": "Axis Labels", "path": "charts-axes-labels" },
+                    { "title": "Grid Lines", "path": "charts-axes-grid-lines" },
+                    { "title": "Secondly Axes", "path": "charts-axes-secondary" }
+                ]
+            },
+            {
+                "title": "Features",
+                "items": [
+                    { "title": "Legend", "path": "charts-legend" },
+                    { "title": "Series Markers", "path": "charts-markers" },
+                    { "title": "Series Highlighting", "path": "charts-series-highlighting" },
+                    { "title": "Tooltips", "path": "charts-tooltips" },
+                    { "title": "Formatters", "path": "charts-formatters" },
+                    { "title": "Cross lines", "path": "charts-axes-cross-lines" },
+                    { "title": "Navigator", "path": "charts-navigator" },
+                    {
+                        "title": "Crosshairs",
+                        "path": "charts-axes-crosshairs",
+                        "isEnterprise": true
+                    },
+                    {
+                        "title": "Context Menu",
+                        "path": "charts-context-menu",
+                        "isEnterprise": true
+                    },
+                    {
+                        "title": "Zoom",
+                        "path": "charts-zoom",
+                        "isEnterprise": true
+                    },
+                    {
+                        "title": "Animation",
+                        "path": "charts-animation",
+                        "isEnterprise": true
+                    }
+                ]
+            },
+            {
+                "title": "AG Grid Integrated Charts",
+                "url": "https://ag-grid.com/javascript-data-grid/integrated-charts/",
+                "isEnterprise": true
+            }
+        ]
+    },
+    "charts": {
```

Comment:
Is this for the gallery or showcase? (It's not clear to me from the name)

Reply:
Neither. It's just for the left hand nav in docs as it's in `menu/data.json`, but it's just for the charts part of the menu (ie, with icons), which is separate from the main part.